src/util/Util.js

```diff
@@ -181,6 +181,10 @@ class Util {
     static setFfmpegPath(path) {
         ffmpeg.setFfmpegPath(path);
     }
+    
+    static sleep(ms) {
+        return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
I rewrite Promises directly in my GroupChat (for some reason) Error: Evaluation failed: ReferenceError: Util is not defined with `const Util = require('../util/Util');`